### DISCUSSION TITLE
Give model creators leaderboard points when their models are used in merges.

### DISCUSSION THREAD

**Discussion by Machinekng** [civitai/civitai#241] [Ideas]
Currently, the leaderboard is ranked based on the models posted by a user. For the purposes of leaderboard scoring, it seems to me that the creator(s) of component models should also receive ranking points when the models they make are merged or have their LORA extracted. That way, creators of original fine-tuned models are highlighted and incentivized to train/post them on civitai.

For example, let's say Alice makes Model A, and Bob merges that model and SD 1.5 to make model B. If model B gets 1000 downloads and 10 likes added to his leader-board score from the model, then Alice should also receive some points on their user ranking. I'm not saying to take points away from Bob and give them to Alice, but just to reward Alice in some proportion. I don't think that it needs to be in the proportion of the contribution. For example, I don't think that Bob using model A as a 15% weighted merge or a 50% weighted merge should make any difference in giving Alice points, since what matters is using models in the correct amounts to have a good effect.

This will obviously require model heritage tracking, but it would be a nice feature to have if heritage is implemented.

**Comment by johndpope** [comment]
good idea.
